keps/NNNN-kep-template/README.md

```diff
@@ -413,6 +417,10 @@ _This section must be completed when targeting beta graduation to a release._
   Longer term, we may want to require automated upgrade/rollback tests, but we
   are missing a bunch of machinery and tooling and do that now.
 
+* **Does the rollout is accompanied with any deprecations of featureds, APIs, fields
```

Comment:
Some typos, change to: "**Is the rollout accompanied by any deprecations of features, APIs, fields".

However, do you mean *deprecations* or *removals*? I think it's removals that are the real issue.

Reply:
Yeah - removals are what we care more about.

I changed to "deprecations and/or removals". Do you think it's good enough?